src/client/datascience/ipywidgets/cdnWidgetScriptSourceProvider.ts

```diff
@@ -99,7 +99,9 @@ export class CDNWidgetScriptSourceProvider implements IWidgetScriptSourceProvide
 
         // Might be on disk, try there first.
         if (diskPath && (await this.fs.localFileExists(diskPath))) {
-            traceInfo(`${ConsoleForegroundColors.Green}Widget Script ${moduleName}#${moduleVersion} found`);
+            traceInfo(
```

Comment:
I thought they needed this logged whether or now we found it? As in they could put the file here when debugging even we don't find it at first?

Reply:
I was thinking of the case where they installed it, but wanted to update. But I could also put a logging just on the initial path search. Probably not a bad idea, I'll add that.